django/db/models/fields/json.py

```diff
@@ -70,6 +70,8 @@ def deconstruct(self):
     def from_db_value(self, value, expression, connection):
         if value is None:
             return value
+        if isinstance(value, dict):
```

Comment:
This is unlikely to be enough as JSON can have an array or string as it's top-level data type.